### DISCUSSION TITLE
请问airtable上的疫情逝者名单如何添加？

### DISCUSSION THREAD

**Discussion by smog150** [The-Run-Philosophy-Organization/run#179] [General]
我不是developer，很少用协同工具，没找到添加按钮。
之前在另一个Google Docs的逝者名单上，我添加过一个身边活生生案例。但没找到airtable上的添加条目按钮？
PS：那个案例的网页快照
https://archive.ph/CmT7l
https://archive.ph/sdYrc

**Comment by ikingye** [comment]
欧美有没有疫情逝者名单？几百万了，google docs 放得下吗？

**Reply by leon-fudan** [reply]
应该还放的下的，一个人占1K字节也没多少。。。。。
最可悲的是，也没人给亮灯默哀降半旗啥的。
就这么默默死了几百万，想想都可怕。

**Comment by smog150** [comment]
> 欧美有没有疫情逝者名单？几百万了，google docs 放得下吗？

欧美不让统计死者名单？

**Comment by the-run-philosophy** [comment]
应该是联系airtable的负责人
